cardano-ledger-core/src/Cardano/Ledger/Era.hs

```diff
@@ -133,6 +138,10 @@ class SupportsSegWit era where
     TxSeq era ->
     Hash.Hash (CryptoClass.HASH (Crypto era)) EraIndependentBlockBody
 
+  -- | Use unsafeApplyTx only for Tests. The real applyTx is an STS rule
+  -- that performs phase 1 validation, as well as the injection into TxInBlock.
+  unsafeApplyTx :: Core.Tx era -> TxInBlock era
```

Comment:
No, this should not live here. If it absolutely is needed, there should be a separate class that exists only in the test libraries

Reply:
done